### DISCUSSION TITLE
Shovel HTTP API: GET /api/shovels/vhost/{vhost}/{name} returns 500 error for names that do not exist

### DISCUSSION THREAD

**Discussion by nesonger** [rabbitmq/rabbitmq-server#12168] [Other] [answered]
I installed RabbitMQ 3.13.7 and was testing the shovel HTTP API: GET /api/shovels/vhost/{vhost}/{name} and noticed that when the shovel name does not exist, I am getting a 500 error instead of the expected 404 or 400 error. 

[errorFromLog.pdf](https://github.com/user-attachments/files/16802668/errorFromLog.pdf)

**Comment by SimonUnge** [comment]
Should probably be an 'issue' and not a discussion. Regardless, easy fix, I'll create a PR.

(FYI @michaelklishin)

**Comment by michaelklishin** [comment, marked as answer]
https://github.com/rabbitmq/rabbitmq-server/pull/12172
